resources/wallet-generator/monero-wallet-generator.html

```diff
@@ -11063,195 +8364,94 @@ <h4 style="color: #555;">Offline wallet</h4>
+                            </p>
+                            <p class="print-only" id="privatekeys_address">Generating...</p>
                         </div>
-                        <div class="col s6">
-                            <input type="button" class="waves-effect waves-light btn btn-qr" onclick="js:toggle_priv_view_qr();" value="Show View Key as QR code" action="" id="show_priv_view_qr_code"/>
+                        <div class="card-action center-align">
+                            <div class="row">
+                                <div class="col s3">
+                                    <span class="label">Spend key:</span>
+                                </div>
+                                <div class="col s9">
+                                    <span id="spend_key_widget">Generating...</span>
+                                    <span class="qrcode" id="priv_spend_qr_widget" onclick="js:toggle_priv_spend_qr();"></span>
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="col s3">
+                                    <span class="label">View key:</span>
+                                </div>
+                                <div class="col s9">
+                                    <span id="view_key_widget">Generating...</span>
+                                    <span class="qrcode" id="priv_view_qr_widget" onclick="js:toggle_priv_view_qr();"></span>
+                                </div>
+                            </div>
+                            <div class="row-new">
+                                <div class="col s6">
+                                    <input type="button" class="waves-effect waves-light btn btn-qr" onclick="js:toggle_priv_spend_qr();" value="Show Spend Key as QR code" action="" id="show_priv_spend_qr_code"/>
+                                </div>
+                                <div class="col s6">
+                                    <input type="button" class="waves-effect waves-light btn btn-qr" onclick="js:toggle_priv_view_qr();" value="Show View Key as QR code" action="" id="show_priv_view_qr_code"/>
+                                </div>
+                            </div>
                         </div>
                     </div>
-                </div>
             </div>
+        <div class="row-new noprint">
+            <h5>4. Verify</h5>
+            <p>
+                This step is optional but highly recommended, as it will allow you to verify that this wallet generator hasn't been tampered in any way.
+            </p>
+            <p>
+                Follow these steps:
+            </p>
+            <ol>
+                <li>Download the signature file: <a href="https://raw.githubusercontent.com/monero-project/monero-site/master/resources/wallet-generator/wallet-generator.html.asc" target="_blank">Download</a></li>
```

Comment:
```suggestion
                <li>Download the signature file: <a href="https://raw.githubusercontent.com/monero-project/monero-site/master/resources/wallet-generator/wallet-generator.html.asc" target="_blank">wallet-generator.html.asc</a></li>
```
I think showing the file name that will be downloaded (or something related) is more standard.